style.css

```diff
@@ -1 +1,56 @@
 /*  Add your styling here */
+body  {
+    display: grid;
+    grid-template-areas: 1fr 2fr 4fr 3r;
```

Comment:
```
    grid-template-areas: 1fr 2fr 4fr 3r;
```
What is this rule trying to do here? Can you take another look?

Reply:
thank you for this, I have removed it as I hadnt actually done anything with the grid and I should have names areas here.